.travis.yml

```diff
@@ -22,6 +22,9 @@ install:
 script: 
   - ./run.sh run_tests
 
+after_success:
+  - R -e 'install.packages("covr")' -e 'covr::codecov()'
```

Comment:
Can you try this alternative which may be quicker as we use the .deb based Travis scheme:

```
after_success:
   - ./run.sh install_aptget r-cran-covr        # available in the Rutter PPAs we know about
   - R -e 'covr::codecov()'
```

That may cut down from the four minutes this currently adds.